documentation/assemblies/assembly-deployment-configuration.adoc

```diff
@@ -18,6 +18,7 @@ This chapter describes how to configure different aspects of the supported deplo
 * Kafka Mirror Maker
 * Kafka Bridge
 * {oauth} token-based authentication
+* {oauth} token-based authorization
```

Comment:
Any reason to have it separately in this list from the OAuth authentication? The OAuth authorization worjks only with OAuth authentication, so I wonder if `{oauth} token-based authentication and authorization` would be better.

Reply:
To keep the layout simple, and also because we kept the authentication content generic, but the authorization is specifically Keycloak. The final section for authentication, _Authorization server examples_, mentions auth server options including Keycloak, which then leads into the next section on authorization. Maybe we live with it like this for now. But I wouldn't mind combining if that's the preference.